CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   ([cyberark/conjur#1456](https://github.com/cyberark/conjur/issues/1456)).
 - Conjur image now performs a `dist-upgrade` as the first image build step to
   ensure the image includes all available vulnerability fixes in the base OS.
+- Upgrade from Rails 4 to Rails 5.2.4.1
+- Upgrade from Cucumber 2 to Cucumber 3.1.2
+- Make significant updates to test and application code so that all tests pass
+  under the new Rails and Cucumber APIs
```

Comment:
I added them based on your suggestion of "things that are likely to cause issues".  I'll remove them since this is your wheelhouse now, but would just like to understand the reasoning.

Reply:
> - Upgrade from Rails 4 to Rails 5.2.4.1

High chance of problems since we're upgrading a significant chunk of our tests and code to accommodate it. It impacts pretty much our whole user-visible codebase.

> - Upgrade from Cucumber 2 to Cucumber 3.1.2

This is a testing framework upgrade. Outside of syntax usages I don't think this will have any user-visible impact and I expect that almost all errors in this changeover will be caught by the CI.

> - Make significant updates to test and application code so that all tests pass
  under the new Rails and Cucumber APIs

This is covered by first two bullet points so it's 100% superfluous.